### PR TITLE
Adding Python Build to Jenkins

### PR DESCRIPTION
**Pull request by kunalgosar**

### CODE REVIEW

File: scripts/jenkins-test [hunk 1]
```diff
@@ -86,6 +86,65 @@ then
...
+PY4J_ZIP="$(ls -1 "${SPARK_HOME}/python/lib" | grep py4j)"
+export PYTHONPATH=${SPARK_HOME}/python:${SPARK_HOME}/python/lib/${PY4J_ZIP}:${PYTHONPATH}
+
+# put gnocchi jar on the pyspark path
+ASSEMBLY_DIR="${Gnocchi_TMP_DIR}/gnocchi-assembly/target"
+ASSEMBLY_JAR="$(ls -1 "$ASSEMBLY_DIR" | grep "^gnocchi[0-9A-Za-z\_\.-]*\.jar$" | grep -v javadoc | grep -v sources || true)"
+export PYSPARK_SUBMIT_ARGS="--jars ${ASSEMBLY_DIR}/${ASSEMBLY_JAR} --driver-class-path ${ASSEMBLY_DIR}/${ASSEMBLY_JAR} pyspark-shell"
+
+# prepare python
+pushd mango-python
```
Comment (nathanielparke) [inline scripts/jenkins-test]: change to gnocchi

File: scripts/jenkins-test [hunk 2]
```diff
@@ -86,6 +86,65 @@ then
...
+uuid=$(uuidgen)
+conda create -q -n gnocchi-build-${uuid} python=2.7 anaconda
+source activate gnocchi-build-${uuid}
+
+# set spark artifact string for downloading assembly
+SPARK=spark-${SPARK_VERSION}
+
+# download prepackaged spark assembly
+wget -q http://d3kbcqa49mib13.cloudfront.net/${SPARK}-bin-${HADOOP}.tgz
+tar xzvf ${SPARK}-bin-${HADOOP}.tgz
```
Comment (nathanielparke) [inline scripts/jenkins-test]: I think the `${SPARK}` and `${HADOOP}` need to be `${SPARK_VERSION}` and `${HADOOP_VERSION}` as those are what are set and checked earlier in the file.

Comment (nathanielparke) [reply scripts/jenkins-test]: Oh wait, I just saw that spark is defined on L#112, do you have to do a similar thing for hadoop?

File: scripts/jenkins-test (hunk 1 above)
Comment (kunalgosar) [reply scripts/jenkins-test]: Sorry this was resolved in: https://github.com/nathanielparke/gnocchi/pull/36/commits/0464bcb6d4749d26f625c6653c86aae2286005bb

File: scripts/jenkins-test (hunk 2 above)
Comment (kunalgosar) [reply scripts/jenkins-test]: resolved

**Pull request merged**